src/dronevis/models/gesture_recognition.py

```diff
@@ -0,0 +1,202 @@
+"""Implementation for gesture recognition using mediapipe"""
+from typing import Union
+import time
+import cv2
+import mediapipe as mp
+import numpy as np
+import torch
+from torch.autograd import Variable
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.optim import *
+import itertools
+import copy
+import os
+from os.path import isfile, join
+
+from dronevis.abstract import CVModel
+from dronevis.utils.general import write_fps
+
+Keypoint_classifier_model_path = join(os.getcwd(),r'src\dronevis\models\hand_keypoint_model_weights.pth')
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+gesture_labels_dict = {"Down":0,"Forward":1, "Left":2, "Right":3, "Stop":4, "Up":5}
```

Comment:
Those variables should be refactored. 
```python
# Capital case since it's a constant
from enum import Enum
KEYPOINT_CLASSIFIER_PATH = join(os.getcwd(),r'src\dronevis\models\hand_keypoint_model_weights.pth') 

# Since values are corresponding to integers, it would be best to use an `enum` here. 
# Values here could be accessed as follows: 'Gesutre.DOWN'.
class Gesture(Enum):
       DOWN = 0
       FORWARD = 1
       LEFT = 2
       RIGHT = 3
       STOP  = 4
       UP = 5

class GestureRecognition(CVModel):
       device: str = "cuda"       
```